### ISSUE TITLE
SEF Plugin breaks style tag

### ISSUE THREAD

**Issue by HLeithner** [joomla/joomla-cms#11264]
#### Steps to reproduce the issue
Insert the following HTML code to your template

```<div id='bg-image' style='background-image:url(database/bg-home.jpg);'></div>```

Show the page with SEF feature on and check the source code.

#### Expected result
```<div id='bg-image' style='background-image:url(database/bg-home.jpg);'></div>```

#### Actual result
```<div id='bg-image' style="background-image: url('/database/bg-home.jpg');'></div>```

#### System information (as much as possible)
In plugins/system/sef/sef.php in Row 153 you find this PHP-Code
```php
		// Replace all unknown protocols in CSS background image.
		if (strpos($buffer, 'style=') !== false)
		{
			$regex  = '#style=\s*[\'\"](.*):\s*url\s*\([\'\"]?(?!/|' . $protocols . '|\#)([^\)\'\"]+)[\'\"]?\)#m';
			$buffer = preg_replace($regex, 'style="$1: url(\'' . $base . '$2$3\')', $buffer);
			$this->checkBuffer($buffer);
		}
```

This will replace style=' with style=" and ignores the old close char '

**Comment by ggppdk** [comment]
I think had noticed this 2 years ago ??  this is old code
and since i was careful not to trigger the bug

@ HLeithner
I have made a PR for it, please test

**Comment by brianteeman** [comment]
Closed as we have a PR for testing #11266<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11264">issues.joomla.org/joomla-cms/11264</a>.</sub>
